### ISSUE TITLE
TypeError: Class constructor SyncedStream cannot be invoked without 'new'

### ISSUE THREAD

**Issue by abaskalov** [maxnowack/meteor-sync-eventemitter#1]
Hello! Iam trying to use your package with rocketchat:streamer.
Iam getting
`SyntaxError: Invalid or unexpected token` near class decor
After adding `babel-plugin-transform-decorators-legacy`
Iam getting `TypeError: Class constructor SyncedStream cannot be invoked without 'new'`

Trying to setup it with your example
```
 import { Meteor } from 'meteor/meteor';
  import sync from 'meteor-sync-eventemitter';

  @sync({ collectionName: 'stream', emitFn: '_emit' })
  class SyncedStream extends Meteor.Streamer {
    constructor() {
      super('streamname');
      this.allowRead( … );
      this.allowWrite( … );
    }
  }

  const stream = new SyncedStream()
  stream.emit( … );
  stream.on( … );
```

![screen shot 2017-12-17 at 18 21 45](https://user-images.githubusercontent.com/3558579/34080968-4d8575c0-e357-11e7-9d3b-c0f6b6747bd8.png)

Could you please give advice how to solve it?

**Comment by maxnowack** [comment]
How does your code in `/imports/startup/both/emitter.js` look like?

**Comment by refresco12** [comment]
I'm having the same issue here.  Is there a recommended .babelrc configuration?

**Comment by maxnowack** [comment]
@rubenpa same question: How does your code look like?

**Comment by refresco12** [comment]
This is in my startup.js:

```
import sync from 'meteor-sync-eventemitter';

@sync({ collectionName: 'stream', emitFn: '_emit' })
class SyncedStream extends Meteor.Streamer {
  constructor() {
    super('game');
    this.allowRead('all');
    this.allowWrite('all');
  }
}

Meteor.startup(function(){
  if (Meteor.isServer) {
    const streamer = new SyncedStream();
    streamer.allowRead('all');
    streamer.allowWrite('all');
  }
});
```

**Comment by maxnowack** [comment]
Can you please also post the error you get?

**Comment by refresco12** [comment]
Same error:

```
TypeError: Class constructor SyncedStream cannot be invoked without 'new'
at new SyncedEventEmitter (/Users/ruben/Documents/projects/node_modules/meteor-sync-eventemitter/dist/syncEventEmitter.js:78:142)
at lib/startup.js:38:22
at Function.time (/Users/ruben/Documents/projects/.meteor/local/build/programs/server/profile.js:309:28)
at /Users/ruben/Documents/projects/.meteor/local/build/programs/server/boot.js:423:13
at /Users/ruben/Documents/projects/.meteor/local/build/programs/server/boot.js:464:5
at Function.run (/Users/ruben/Documents/projects/.meteor/local/build/programs/server/profile.js:510:12)
at /Users/ruben/Documents/projects/.meteor/local/build/programs/server/boot.js:462:11
```

**Comment by refresco12** [comment]
Any idea on what might be causing this?

**Comment by maxnowack** [comment]
Sorry, I don't have much time at the moment. Have you tried to use it without a ES7 decorator? e.g. like in the tests: https://github.com/maxnowack/meteor-sync-eventemitter/blob/master/tests/imports/sync.test.js#L10

**Comment by maxnowack** [comment]
fixed in `v1.0.1`

**Comment by refresco12** [comment]
Thanks for taking the time with this!  Will try it out soon
